skynet/ipfs/docker.py

```diff
@@ -63,6 +63,8 @@ def open_ipfs_node(
         if ec != 0:
             logging.error(out)
 
+    # TODO, why not deliver some kinda API here for controlling the
+    # ipfs node?
```

Comment:
I like the idea

Reply:
Ah wait! this is actually part of our old ipfs node launch system, we try to not manage the IPFS node from tihs codebase and provide a standard template at https://github.com/skygpu/ipfs-node-template